app/_components/nav-links.tsx

```diff
@@ -26,16 +30,19 @@ export default function NavLinks() {
             key={link.name}
             href={link.href}
             className={clsx(
-              `text-black text-lg font-medium text-slate-800 hover:text-lime-600 grow mx-38 ${inter.className} `,
+              styles.headerNavigation,
               {
-                "text-lime-600": pathname === link.href,
+                "text-lime-600 border-b border-lime-500": pathname === link.href,
```

Comment:
if you plan to keep tailwind - this is fine.
Otherwise please add your own styles.